mlir/include/mlir/Dialect/Arith/IR/ArithOps.td

```diff
@@ -1195,6 +1195,12 @@ def Arith_ExtFOp : Arith_FToFCastOp<"extf"> {
   }];
   let hasVerifier = 1;
   let hasFolder = 1;
+
+  let arguments = (ins FloatLike:$in, OptionalAttr<Arith_FastMathAttr>:$fastmath);
```

Comment:
I'd go with `DefaultValuedAttrArith_FastMathAttr,` "::mlir::arith::FastMathFlags::none">` to match stuff like `Arith_UnaryFloatOp`

Reply:
changed to default value.